src/app/list/[slug]/[id]/page.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+import React from 'react';
+import { useEffect, useState } from 'react';
+import { ListItem } from '@/types/apis/listItem';
+import PageTitle from '@/components/PageTitle/PageTitle';
+import * as S from './listDetail.style';
```

Comment:
이거 아직 규칙 안정하셨을 수도 있는데 import 순서... 조금 신경쓰이네요
```suggestion
import PageTitle from '@/components/PageTitle/PageTitle';
import { ListItem } from '@/types/apis/listItem';
import * as S from './listDetail.style';
```

Reply:
일반적으로 사용하는 import 순서 컨밴션이 있을까요?
규칙 정하진 않았지만 제가 생각했던 건 아래 순서였습니다..!
1. react 컴포넌트 및 훅
2. 외부 라이브러리
3. 내부 모듈 or 컴포넌트
5. 스타일

Reply:
그렇게 사용하시면 됩니다!